tests/output/t302-basis-f.out

```diff
@@ -1,48 +1,11 @@
-CeedBasis: dim=1 P=4 Q=4
-      qref1d:	 -1.00000000	 -0.44721360	  0.44721360	  1.00000000
-   qweight1d:	  0.16666667	  0.83333333	  0.83333333	  0.16666667
-    interp1d[0]:	  1.00000000	  0.00000000	  0.00000000	  0.00000000
-    interp1d[1]:	  0.00000000	  1.00000000	  0.00000000	  0.00000000
-    interp1d[2]:	  0.00000000	  0.00000000	  1.00000000	  0.00000000
-    interp1d[3]:	  0.00000000	  0.00000000	  0.00000000	  1.00000000
-      grad1d[0]:	 -3.00000000	  4.04508497	 -1.54508497	  0.50000000
-      grad1d[1]:	 -0.80901699	  0.00000000	  1.11803399	 -0.30901699
-      grad1d[2]:	  0.30901699	 -1.11803399	  0.00000000	  0.80901699
-      grad1d[3]:	 -0.50000000	  1.54508497	 -4.04508497	  3.00000000
 collograd[0]: -3.00000000  4.04508497 -1.54508497  0.50000000
```

Comment:
Are the `out` files supposed to be pushed?

Reply:
Yes, the out files are the comparison files for the tests that intentionally write output.

Reply:
Yes, thanks. I had just misread the path. The question was meant to ask about being "pushed here" and I typed too fast